pages/common/paci.md

```diff
@@ -0,0 +1,23 @@
+# paci
+
+> Your friendly, lightweight and flexible package manager. Written in Python.
+
+- Install `paci` and configure it:
```

Comment:
tldr pages should explain only the command usage, not how to install a tool. Let us remove this example

Reply:
Ah okay. I'll replace it with the command to how to configure it.